ddtrace/settings/_config.py

```diff
@@ -626,6 +626,7 @@ def __init__(self):
             "DD_CIVISIBILITY_EARLY_FLAKE_DETECTION_ENABLED", True, asbool
         )
         self._otel_enabled = _get_config("DD_TRACE_OTEL_ENABLED", False, asbool, "OTEL_SDK_DISABLED")
+        self._otel_metrics_enabled = _get_config("DD_TRACE_OTEL_METRICS_ENABLED", False, asbool, "OTEL_SDK_DISABLED")
```

Comment:
```suggestion
        self._otel_metrics_enabled = _get_config("DD_TRACE_OTEL_METRICS_ENABLED", False, asbool, "OTEL_SDK_DISABLED")
```

Should we use DD_OTEL_METRICS_ENABLED instead? This isn't a tracing feature 😅

Reply:
My idea for combining the two right now is `DD_TRACE_EXPERIMENTAL_FEATURES_ENABLED=DD_METRICS_OTEL_ENABLED`. WDYT? This environment variable name may change during the RFC which is why I think hiding it behind the `DD_TRACE_EXPERIMENTAL_FEATURES_ENABLED` configuration seems appropriate

Reply:
As long as we don't document this configuration then we don't need to make a final decision right now. `DD_TRACE_EXPERIMENTAL_FEATURES_ENABLED=DD_METRICS_OTEL_ENABLED` sounds good to me.